tests/hawc/apps/animal/test_models.py

```diff
@@ -3,6 +3,13 @@
 from hawc.apps.animal import models
 
 
+@pytest.mark.django_db
+class TestAnimalGroup:
+    def test_can_delete(self):
```

Comment:
would you mind writing a test here? I'm imagining you'd create a new experiment (repro), and then a parent w/ dosing regime and a child w/o. Then you'd run through these checks

```python
assert parent.can_delete() is False
assert child._can_delete() is True
child.delete()
assert parent.can_delete() is True
```

Reply:
I ended up testing the parent/child case separately from the no dosing/regime case, since the parent can still be deleted if none of its children have a dosing regime